packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/SubTaskPopup/SubTaskPopup.js

```diff
@@ -1,19 +1,64 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { PropTypes as MobXPropTypes } from 'mobx-react'
-import { observer } from 'mobx-react'
+import counterpart from 'counterpart'
 import { Box, Paragraph } from 'grommet'
-import { MovableModal } from '@zooniverse/react-components'
-import SaveButton from './components/SaveButton'
-import getDefaultPosition from '../../helpers/getDefaultPosition'
+import { observer, PropTypes as MobXPropTypes } from 'mobx-react'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { Modal, MovableModal, PrimaryButton } from '@zooniverse/react-components'
+
 import taskRegistry from '@plugins/tasks'
+import getDefaultPosition from '../../helpers/getDefaultPosition'
+import SaveButton from './components/SaveButton'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
 
 const MIN_POPUP_WIDTH = 350
 const MIN_POPUP_HEIGHT = 100
 
+export const StyledPrimaryButton = styled(PrimaryButton)`
+  background: none;
+  ${props => props.theme.dark ? 
+    css`color: ${props.theme.global.colors['light-3']};` :
+    css`color: ${props.theme.global.colors['dark-3']};`
+  }
+`
+
+export function ConfirmModal({ onClose, onDelete }) {
+  return (
+    <Modal
+      active
+      closeFn={onClose}
+    >
+      <Paragraph
+        margin={{ bottom: 'medium' }}
+      >
+        {counterpart('Task.confirm')}
+      </Paragraph>
+      <Box
+        direction='row'
+        gap='small'
+        justify='center'
+      >
+        <StyledPrimaryButton
```

Comment:
This can be a standard Grommet Button (defaults to the brand teal already w/ no background color):

<img width="444" alt="Screen Shot 2020-09-24 at 2 27 01 PM" src="https://user-images.githubusercontent.com/5016731/94190812-3b218200-fe72-11ea-802b-71d6df394649.png">